lib/grammers-session/src/message_box/adaptor.rs

```diff
@@ -505,6 +509,22 @@ impl PtsInfo {
             DeleteQuickReply(_) => None,
             QuickReplyMessage(_) => None,
             DeleteQuickReplyMessages(_) => None,
+            BotBusinessConnect(_) => None,
+            BotNewBusinessMessage(u) => Some(Self {
+                pts: u.qts,
+                pts_count: 0,
+                entry: Entry::SecretChats,
+            }),
+            BotEditBusinessMessage(u) => Some(Self {
+                pts: u.qts,
+                pts_count: 0,
```

Comment:
Similarly, this probably also needs to be 1 because there is 1 message affected.